rocketchat/Chart.yaml

```diff
@@ -1,12 +1,16 @@
 apiVersion: v2
 name: rocketchat
-version: 4.5.4
-appVersion: 4.5.4
+version: 4.6.0
+appVersion: 4.6.0
```

Comment:
I think we should leave it a 4.5.4, for now. If it's 4.6.0 it fails to pull the correct Docker images. I've tried using 4.6.0-rc1 here, as well, and it too didn't work. 4.5.4, on the other hand worked like a charm.